WcaOnRails/app/webpacker/components/CompetitionOverview/CompetitionTable.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import {
+  List, Icon, Popup, Loader,
+} from 'semantic-ui-react';
+
+import I18n from '../../lib/i18n';
+import { dayDifferenceFromToday, PseudoLinkMarkdown } from '../../lib/utils/competition-table';
+
+function CompetitionTable({
+  competitionData,
+  title,
+  shouldShowRegStatus,
+  shouldIncludeCancelled,
+  selectedEvents,
+  isLoading,
+  hasMoreCompsToLoad,
+  isSortedByAnnouncement = false,
+  isRenderedAboveAnotherTable = false,
+}) {
+  const competitions = competitionData?.filter((comp) => (
+    (!comp.cancelled_at || shouldIncludeCancelled)
+    && (selectedEvents.every((event) => comp.event_ids.includes(event)))
+  ));
+
+  return (
+    <List divided relaxed>
+      <List.Item>
+        <strong>
+          {`${title} (${competitions ? competitions.length : 0}${hasMoreCompsToLoad ? '...' : ''})`}
+        </strong>
+      </List.Item>
+      {competitions?.map((comp, index) => (
+        <React.Fragment key={comp.id}>
+          <ConditionalYearHeader
+            competitions={competitions}
+            index={index}
+            isSortedByAnnouncement={isSortedByAnnouncement}
+          />
+          <List.Item className={`${comp.isProbablyOver ? ' past' : ' not-past'}${comp.cancelled_at ? ' cancelled' : ''}`}>
+            <span className="date">
+              <StatusIcon
+                comp={comp}
+                shouldShowRegStatus={shouldShowRegStatus}
+                isSortedByAnnouncement={isSortedByAnnouncement}
+              />
+              {comp.dateRange}
+            </span>
+            <span className="competition-info">
+              <div className="competition-link">
+                <span className={` fi fi-${comp.country_iso2}`} />
+                &nbsp;
+                <a href={comp.url}>{comp.displayName}</a>
+              </div>
+              <div className="location">
+                <strong>{comp.countryName}</strong>
+                {`, ${comp.cityName}`}
+              </div>
+              <div className="venue-link">
+                <PseudoLinkMarkdown text={comp.venue} />
+              </div>
+            </span>
+          </List.Item>
+        </React.Fragment>
+      ))}
+      {/* Could not figure out why Semantic UI's animated loader icon doesn't show */}
```

Comment:
Is this still an issue? Maybe it will work, or be easier to debug, given all the recent refactoring?

Reply:
Yeah there should be an animated circle rather than just the "loading..." text. I suspect it's something strange with CSS but just couldn't pinpoint the issue after having tried for a while.

Reply:
Turns out I just needed to import semantic UI's css. Thought that was already done!